db_diff/tskdbdiff.py

```diff
@@ -999,28 +1009,20 @@ def normalize_tsk_objects_path(guid_util: TskGuidUtils, objid: int,
                 path_parts = path_parts[:-1]
 
         if path_parts and len(path_parts) >= 2:
-            is_leapp = False
```

Comment:
I think we need the leapp path sanitization for some of our current testing.  It removes some timestamp path segments which change daily.